client/src/components/ModelGraphViewerComponent/ModelGraphViewerCytoscapeComponent/ModelGraphViewerCytoscapeComponent.js

```diff
@@ -153,7 +158,30 @@ export class ModelGraphViewerCytoscapeComponent extends React.Component {
     return settingsService.getModelImage(modelId);
   }
 
+  saveLayout() {
+    storageService.saveModelsLayoutPositions(sessionService.getModelsLayoutPositions());
+    const msg = `${this.layout} layout saved`;
+    print(msg);
+    this.setState({ notificationMessageIsVisible: true, notificationMessage: msg });
+  }
+
+  clearLayout() {
+    sessionService.clearModelsLayout(this.layout);
+    storageService.clearModelsLayout(this.layout);
```

Comment:
As per earlier comment, we're saving in two places?

Reply:
Created LayoutService class to handle the calls of each respective component in a unified class